leiden-journal-of-international-law.csl

```diff
@@ -0,0 +1,637 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="en-GB" and="symbol" et-al-min="3" initialize-with="." sort-separator=" " initialize-with-hyphen="false" demote-non-dropping-particle="never">
+  <info>
+    <title>Leiden Journal of International Law</title>
+    <title-short>LJIL</title-short>
+    <id>"http://www.zotero.org/styles/leiden-journal-of-international-law.csl</id>
+    <link href="http://www.zotero.org/styles//leiden-journal-of-international-law.csl" rel="self"/>
+    <link href="https://www.cambridge.org/core/services/aop-file-manager/file/575adb7dfcaa6b122ec738b3" rel="documentation"/>
+    <author>
+      <name>Øyvind Ege</name>
+      <email>oyvind.ege@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="international law"/>
+    <issn>0922-1565</issn>
+    <eissn>1478-9698</eissn>
+    <summary>Citation as per LJIL standards</summary>
+    <published>May 2017</published>
```

Comment:
This has to be a specially formatted timestamp, like the one you used for `<updated/>` (per http://books.xmlschemata.org/relaxng/ch19-77049.html). But we almost never use `<published/>` in this repository, so you can also just delete this line.